### PR TITLE
Added piped instance urls

### PR DESCRIPTION
**Pull request by JordanWeatherby**
### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

---

### Description of your *pull request* and other information

Adds urls for other instances of piped as outlined in https://github.com/yt-dlp/yt-dlp/issues/3661

Didn't add tests as the insidious instances in the same file do not have tests, have tested and all instances work however

### CODE REVIEW

File: yt_dlp/extractor/youtube.py [hunk 1]
```diff
@@ -348,6 +348,16 @@ class YoutubeBaseInfoExtractor(InfoExtractor):
         r'(?:www\.)?kbjggqkzv65ivcqj6bumvp337z6264huv5kpkwuv6gu5yjiskvan7fad\.onion',
         r'(?:www\.)?grwp24hodrefzvjjuccrkw3mjq4tzhaaq32amf33dzpmuxe7ilepcmad\.onion',
         r'(?:www\.)?hpniueoejy4opn7bc4ftgazyqjoeqwlvh2uiku2xqku6zpoa4bf5ruid\.onion',
+        # piped instances
```
Comment (pukkandan) [inline yt_dlp/extractor/youtube.py:351]:
```suggestion
        # piped instances from https://github.com/TeamPiped/Piped/wiki/Instances
```

Make sure to add all these

Comment (JordanWeatherby) [reply yt_dlp/extractor/youtube.py]: Added the ones that work (piped.notyourcomputer.net seems to redirect to the github and not an instance)

**Comment by jakoblm** [comment]
Would you also add the following instance?
https://piped.sp-codes.de/

**Comment by pc00per** [comment]
Instances are closing & opening every month so far. In this page, yt-dlp kinda can't keep up with em. Some form of other workaround needs to be discovered.
I'm currently using https://api-piped.shimul.me/ & it doesn't work. But there are lot more others, refer https://github.com/TeamPiped/Piped/wiki/Instances

**Pull request merged**